.github/workflows/stale.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
     - uses: actions/stale@v5
       with:
-        repo-token: ${{ secrets.STALE_PERSONAL_ACCESS_TOKEN }}
+        repo-token: ${{ github.token }}
```

Comment:
If it’s default, no need to set it again explicitly.
`repo-token: ${{ github.token }}`  can be removed.

Also, stale action is at v9 now.